packages/react/src/components/button.tsx

```diff
@@ -146,7 +146,7 @@ export function UploadButton<TRouter extends FileRouter>(
     >
       <label
         className={twMerge(
-          "relative flex h-10 w-36 cursor-pointer items-center justify-center overflow-hidden rounded-md text-white after:transition-[width] after:duration-500",
+          "relative flex h-10 w-36 cursor-pointer items-center justify-center overflow-hidden rounded-md text-white after:transition-[width] after:duration-500 focus-within:ring-blue-600 focus-within:ring-offset-2",
```

Comment:
I don't see this focus state: 

https://github.com/pingdotgg/uploadthing/assets/51714798/d52f404e-95cc-4919-a328-e1fc42b9d25d

Reply:
just had to add the ring class too